linkerd2-proxy/src/main.rs

```diff
@@ -5,18 +5,27 @@
 #![type_length_limit = "1110183"]
 
 use linkerd2_app::Main;
-use linkerd2_signal as signal;
+use linkerd2_signal as sig;
 
-/// Loads configuration from the environment
 fn main() {
-    let (config, trace_admin) = match linkerd2_app::init() {
-        Ok(c) => c,
+    // Load configuration from the environment without binding ports.
+    let main = match Main::try_from_env() {
+        Ok(main) => main,
         Err(e) => {
-            eprintln!("configuration error: {:#?}", e);
-            std::process::exit(64)
+            eprintln!("Configuration error: {}", e);
+            std::process::exit(64) // EX_USAGE
```

Comment:
Nit/TIOLI: if you add a const, this could be `std::process:exit(EX_USAGE);` and the comment wouldn't be necessary...not a big deal though.